### PR TITLE
Move base image to Alpine

### PR DESCRIPTION
**Pull request by cjkoenig**
I changed your Dockerfile to use Alpine 3.4 and the image is now shrinked to ~95MB (#1). Since I had to change the scripts and have (so far...) no AWS account, can you please test if it is working as expected?

### CODE REVIEW

**Comment by grahovam** [comment]
You're awesome, thanks. I'll check it asap

File: run.sh [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
+# Import GPG public keys
 gpg --import /keys/*
 
-cron
-
-# LS_COLORS is set to nothing and for some strange reason crontabs are not allowed to contain such env vars
-unset LS_COLORS
-
-# Create crontab file
-env | cat - > /backup.cron
```
Comment (grahovam) [inline run.sh:11]: Why did you remove the environment variables here?

Comment (cjkoenig) [inline run.sh:11]: LS_COLORS is not set in Alpine and the environment variables should be available to cron...aren't they?

Comment (grahovam) [inline run.sh:11]: env vars are not available to cron. That's why you have to explicitly add them with
`env | cat - > /backup.cron`

Comment (cjkoenig) [inline run.sh:11]: For me, the environment variables are available in backup.sh called by cron.

To verify that, I did the following in the switch_to_alpine branch:
- add `echo "S3_BUCKET_NAME: $S3_BUCKET_NAME" >> /test.txt` to backup.sh
- build and run the container with the command from your README.md
- make sure the crontab is installed correctly: `docker exec my-backup crontab -l`
- output: `0 4 * * *  /backup.sh`
- run `docker exec my-backup sh /backup.sh`
- run `docker exec my-backup cat test.txt`
- output: `S3_BUCKET_NAME: myBackupBucket`
